.github/workflows/docs.yml

```diff
@@ -33,6 +33,16 @@ jobs:
 
     if: github.event_name != 'pull_request' || !contains(github.event.pull_request.labels.*.name, 'no ci')
 
+    on:
+      push:
+      pull_request:
+        paths_ignore:
+          - 'cmd/**'
+          - 'ferretdb/**'
+          - 'integration/**'
+          - 'internal/**'
+          - 'tools/**'
```

Comment:
I think the whole block should be above, where we already have `on.pull_request`